sdks/java/io/elasticsearch/src/main/java/org/apache/beam/sdk/io/elasticsearch/ElasticsearchIO.java

```diff
@@ -116,6 +116,18 @@
 @Experimental(Experimental.Kind.SOURCE_SINK)
 public class ElasticsearchIO {
 
+  private static void checkVersion(ConnectionConfiguration connectionConfiguration)
```

Comment:
Move it back to keep the history (And, I think private methods are not that important to be on the top.)